### PR TITLE
net-misc/proxychains: add 4.17, EAPI8, proxy-maint

### PR DESCRIPTION
**Pull request by hlein**
Closes: https://bugs.gentoo.org/911140

### CODE REVIEW

File: net-misc/proxychains/proxychains-4.17.ebuild [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+KEYWORDS="~amd64 ~ppc ~riscv ~sparc ~x86"
+
+S=${WORKDIR}/${MY_P}
+
+PATCHES=( "${FILESDIR}"/${P}-makefile.patch )
+
+src_prepare() {
+	default
+	sed -i "s/^\(LDSO_SUFFIX\).*/\1 = so.${PV}/" Makefile || die
+	mv completions/zsh/_proxychains4 completions/zsh/_proxychains
```
Comment (dlan17) [inline net-misc/proxychains/proxychains-4.17.ebuild:26]: mv may fail, so should add "|| die"

**Comment by dlan17** [comment]
I've also got multi definition error, please check full build.log here
https://dev.gentoo.org/~dlan/logs/build.log

**Comment by hlein** [comment]
> I've also got multi definition error, please check full build.log here https://dev.gentoo.org/~dlan/logs/build.log

Thanks, I can reproduce this here now too. Haven't figure out why yet... nor why i *didn't* hit this a week ago (have tested rolling back any updates to the usual suspects toolchain wise, etc.). Still investigating...

**Comment by dlan17** [comment]
I've spent some time to investigate this, and found there was a problem in proxychains-4.17-makefile.patch which cause object files been linked twice, I will fix this & push it, thanks

**Comment by dlan17** [comment]
Author: Hank Leininger <hlein@korelogic.com>
Date:   Wed Feb 14 21:15:33 2024 -0700

    net-misc/proxychains: add 4.17, EAPI8, proxy-maint

    Closes: https://bugs.gentoo.org/911140
    Closes: https://github.com/gentoo/gentoo/35339
    Signed-off-by: Hank Leininger <hlein@korelogic.com>
    Signed-off-by: Yixun Lan <dlan@gentoo.org>

**Comment by hlein** [comment]
> I've spent some time to investigate this, and found there was a problem in proxychains-4.17-makefile.patch which cause object files been linked twice, I will fix this & push it, thanks

Woo, thank you!

**Pull request closed without merging**